src/input/SimModSuite.h

```diff
@@ -357,6 +357,54 @@ struct get_second : public std::unary_function<MyMap::value_type, int>
     }
 };
 
+enum elementType {
+  vertex = 0,
+  edge = 1,
+  face = 2,
+  region = 3
+};
+
+void setMeshSize(pGModel model, pACase &meshCase, MeshAttributes &MeshAtt) {
+    // Set mesh size on vertices, edges, surfaces and regions
+    const char *element_name[4] = { "vertex", "edge", "face", "region" };
+
+    for (int elType = vertex; elType != region+1; elType++ )
+    {
+        std::list <std::pair <std::list<int>, double>> lpair_lId_MSize;
+        switch(elType) {
+        case vertex:
+            lpair_lId_MSize = MeshAtt.lpair_lVertexId_MSize;
+            break;
+        case edge:
+            lpair_lId_MSize = MeshAtt.lpair_lEdgeId_MSize;
+            break;
+        case face:
+            lpair_lId_MSize = MeshAtt.lpair_lFaceId_MSize;
+            break;
+        case region:
+            lpair_lId_MSize = MeshAtt.lpair_lRegionId_MSize;
+            break;
+        }
+
+        for (auto itr = lpair_lId_MSize.begin(); itr != lpair_lId_MSize.end(); itr++)
```

Comment:
Maybe you can give the iterators more verbose names? `itr` and `it` sound pretty much the same.

Reply:
I ve refactor this code.